Example/src/DragAndSnapExample.tsx

```diff
@@ -1,21 +1,31 @@
-import React from 'react';
+import React, { FC } from 'react';
 import { View } from 'react-native';
 import Animated, {
   useSharedValue,
   withSpring,
   useAnimatedStyle,
   useAnimatedGestureHandler,
   interpolate,
+  Extrapolate,
 } from 'react-native-reanimated';
-import { PanGestureHandler } from 'react-native-gesture-handler';
+import {
+  PanGestureHandler,
+  PanGestureHandlerGestureEvent,
+} from 'react-native-gesture-handler';
 
-function DragAndSnap() {
+const DragAndSnap: FC = () => {
   const translation = {
     x: useSharedValue(0),
     y: useSharedValue(0),
   };
-
-  const gestureHandler = useAnimatedGestureHandler({
+  type AnimatedGHContext = {
+    startX: number;
+    startY: number;
+  };
```

Comment:
I see that you define `AnimatedGHContext` type many times. Does gesture-handler not contains this type?

Reply:
I _could_ use some type from RNGH but context is distinct from RNGH event payload so I think it's better defining different interface than using `Pick<PanGestureHandlerGestureEvent, "startX", "startX">`.